chore(release): Automated agent release notes (v0.6.12) (#1088)
armory/docs: content/en/armory-enterprise/release-notes/rn-armory-agent/agent-service/agent-service-v-0-6-12.md

---

> Maintenance to update the directory where the agent releases notes has been store

Reviewer: ```suggestion
## Changes

Update the directory where the Agent releases notes are stored.

```
